### PR TITLE
Updates version of JSDOM to latest (v11) Fixes "Click" Tests which means 100% Coverage!!! 

### PR DESCRIPTION
**Pull request by nelsonic**
Fixes:
+ JSDOM Version: https://github.com/dwyl/learn-elm-architecture-in-javascript/issues/27
+ (Server-side) "Click" Tests: https://github.com/dwyl/learn-elm-architecture-in-javascript/issues/20

### CODE REVIEW

**Comment by nelsonic** [comment]
I _know_ I'm a "_freak_" but the "***95% Coverage***" Was ***Bugging*** me!!
image

[image](https://youtu.be/btO3kE2RrEY)
https://youtu.be/btO3kE2RrEY

@iteles please review/merge when you can. thanks!

**Comment by nelsonic** [comment]
@iteles review/merge not "urgent". but would be nice to get the "Tweet" done on Monday. 😉

File: test/test.js [hunk 1]
```diff
@@ -81,50 +82,54 @@ function(assert) {
...
+test('Click reset button resets state to 0', function(assert) {
+  mount(7, update, view, id);
+  var root = document.getElementById(id);
+  assert.equal(root.getElementsByClassName('count')[0].textContent, 7);
+  var btn = root.getElementsByClassName("reset")[0]; // click reset button
+  btn.click(); // Click the Reset button!
+  var state = root.getElementsByClassName('count')[0].textContent;
+  assert.equal(state, 0, "State is 0 (Zero) after reset."); // state was successfully reset to 0!
+  empty(root); // clean up after tests
+  // console.log('STATE:', state);
```
Comment (iteles) [inline test/test.js]: There are a couple of commented out `console.logs` but I'm happy to leave them in for the time being as these might actually be useful to beginners when we publicise the tutorial #29

File: test/test.js [hunk 2]
```diff
@@ -81,50 +82,54 @@ function(assert) {
...
-//   var result = update(6, "reset");
-//   assert.equal(result, 0);
-// });
-//
+test('Test reset counter when model/state is 6 returns 0', function(assert) {
+  var result = update(6, "reset");
+  assert.equal(result, 0);
+});
+
+// please make this test pass if you know how! thanks! :-)
```
Comment (iteles) [inline test/test.js]: Worth also opening an issue on this 👍

**Comment by iteles** [comment]
I love how you hearted the codecov comment here 😆

File: test/test.js (hunk 2 above)
Comment (nelsonic) [inline test/test.js]: please feel free to open that issue. thanks. 👍

**Pull request merged**